diff --git a/dist/types.d.ts b/dist/types.d.ts
index 4461847916ceaa76f3d120b802e543fd0597c053..a5eb168a1e580f1d04192bd23f6c6730669979a3 100644
--- a/dist/types.d.ts
+++ b/dist/types.d.ts
@@ -10,7 +10,7 @@ type PropsWeControl = 'as' | 'children' | 'refName' | 'className';
 type CleanProps<TTag extends ReactTag, TOmitableProps extends PropertyKey = never> = Omit<PropsOf<TTag>, TOmitableProps | PropsWeControl>;
 type OurProps<TTag extends ReactTag, TSlot> = {
     as?: TTag;
-    children?: ReactNode | ((bag: TSlot) => ReactElement);
+    children?: any;
     refName?: string;
 };
 type HasProperty<T extends object, K extends PropertyKey> = T extends never ? never : K extends keyof T ? true : never;