diff --git a/dist/types.d.ts b/dist/types.d.ts
index 5d4d0bc1c64e5abb7eaf1f54db31e13558ec9330..63146fdcc3cf22d78e9d16923e1f2fa12770771e 100644
--- a/dist/types.d.ts
+++ b/dist/types.d.ts
@@ -10,7 +10,7 @@ declare type PropsWeControl = 'as' | 'children' | 'refName' | 'className';
 declare type CleanProps<TTag extends ReactTag, TOmitableProps extends PropertyKey = __> = TOmitableProps extends __ ? Omit<PropsOf<TTag>, PropsWeControl> : Omit<PropsOf<TTag>, TOmitableProps | PropsWeControl>;
 declare type OurProps<TTag extends ReactTag, TSlot> = {
     as?: TTag;
-    children?: ReactNode | ((bag: TSlot) => ReactElement);
+    children?: any;
     refName?: string;
 };
 declare type HasProperty<T extends object, K extends PropertyKey> = T extends never ? never : K extends keyof T ? true : never;