diff --git a/es/components/Provider.d.ts b/es/components/Provider.d.ts
index 4a51c0f825cfef75fe1fd855bb863a1e5fd76607..257ba8ba0d00220c89d9f20759e9382da5a97dd3 100644
--- a/es/components/Provider.d.ts
+++ b/es/components/Provider.d.ts
@@ -16,7 +16,7 @@ export interface ProviderProps<A extends Action = AnyAction, S = unknown> {
      * Initial value doesn't matter, as it is overwritten with the internal state of Provider.
      */
     context?: Context<ReactReduxContextValue<S, A>>;
-    children: ReactNode;
+    children: any;
 }
 declare function Provider<A extends Action = AnyAction, S = unknown>({ store, context, children, serverState, }: ProviderProps<A, S>): JSX.Element;
 export default Provider;